tests/test_iotedgedev.py

```diff
@@ -125,11 +129,17 @@ def test_module_add():
     add_module_and_verify(cli.main, runner, "c")
     add_module_and_verify(cli.main, runner, "csharp")
     add_module_and_verify(cli.main, runner, "java")
-    add_module_and_verify(cli.main, runner, "nodejs")
+
+    if (platform.system().lower() == 'windows'):
```

Comment:
Please add link to the GitHub issue as comment so that following maintainer knows why adding Node.js modules is skipped on non-Windows.

Reply:
Updated code